core/tests/mad_hatter/test_mad_hatter.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from cat.mad_hatter.mad_hatter import MadHatter
+from cat.looking_glass.cheshire_cat import CheshireCat
+
+
+from unittest.mock import MagicMock
+from unittest.mock import patch
+
+def test_install_plugin():
+    ccat = CheshireCat()
+    ccat.get_plugin_path = MagicMock(return_value="./tests/mad_hatter/plugin_folder")
```

Comment:
Love that you mock a different plugin folder (I'll update to do it also [here](https://github.com/cheshire-cat-ai/core/blob/main/core/tests/routes/test_routes_plugins.py#L33)).
Question: we are using `pytest` for testing, here you use utilities from `unittest`. Can we do the mocks [as in here](https://github.com/cheshire-cat-ai/core/blob/05e97cc569128dbdf2580bf95fc65f8738169de5/core/tests/conftest.py#L33) or are we forced to combine the two libraries?